tests/phpunit/tests/template.php

```diff
@@ -82,14 +82,6 @@ public function setUp() {
 		$this->set_permalink_structure( '/%year%/%monthnum%/%day%/%postname%/' );
 	}
 
-	public function tearDown() {
-		unregister_post_type( 'cpt' );
-		unregister_taxonomy( 'taxo' );
-		$this->set_permalink_structure( '' );
```

Comment:
Similar remark as before: the fact that these are (conditionally) reset in `setUp()` does not negate the need to reset in `tearDown()`.